src/ui/backend/BaseBacking.js

```diff
@@ -62,36 +83,39 @@ export default class BaseBacking {
 
   copy () {
     var copy = {};
+    for (var key in BASE_STYLE_PROPS) {
+      copy[key] = this[key];
+    }
 
-    for (var i = 0; i < styleKeyList.length; i++) {
-      var key = styleKeyList[i];
+    for (var i = 0; i < extendedStylePropList.length; i++) {
+      var key = extendedStylePropList[i];
       copy[key] = this[key];
     }
 
     return copy;
   }
 
   update (style) {
-    for (var i = 0; i < styleKeyList.length; i++) {
-      var key = styleKeyList[i];
+    for (var key in style) {
+      if (this[key] !== undefined) {
+        this[key] = style[key];
+      }
+    }
+
+    for (var i = 0; i < extendedStylePropList.length; i++) {
+      var key = extendedStylePropList[i];
       if (style[key] !== void 0) {
         this[key] = style[key];
       }
     }
+
     return this;
   }
 
 };
 
-var styleKeys = BaseBacking.prototype.constructor.styleKeys = {};
-var styleKeyList = [];
-
+var extendedStylePropList = [];
```

Comment:
My two cents is that if we're breaking APIs, then we should just have a LayoutView that extends View and be done with it.